packages/react-components/src/label/src/label.jsx

```diff
@@ -42,6 +42,10 @@ const propTypes = {
      * Whether to add emphasis on the label text or not.
      */
     highlight: bool,
+    /**
+     * @ignore
+     */
+    disabled: bool,
```

Comment:
You don't need this since the tag component is using the SUI Label and not this custom one.